css/style.css

```diff
@@ -0,0 +1,172 @@
+
+body {
+  margin-left: 50px;
+  margin-right: 50px;
+  background-color: #eeeeee;
+
+}
+
+@media only screen and (max-width: 600px) {
+  body {
+    background-color: #eeeeee;
+  }
+}
+#header{
+    position: fixed;
+    top: 0;
+    min-height: 75px;
+    padding: 0px 20px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: #eee;
+    width: 100%;
+}
+
+.header-img {
```

Comment:
Kindly set up linters in your project ( Webhint , Stylelint ). You can find the reference in the guide we provided you with in your first block.